src/lib/action.rs

```diff
@@ -267,6 +335,40 @@ impl<T: Clone + Debug + Eq + 'static> Patchify<T> for Update<T> {
     }
 }
 
+impl<T: Clone + Debug + Eq + 'static> Patchify<T> for Copy{
+    fn patchify(&self, store: &MappingStore<T>, from: &mut Vec<Patch>, to: &mut Vec<Patch>) {
```

Comment:
OK, the `patchify` trait looks OK. This is something for us to think about later down the line, because `patchify` takes a `MappingStore` as an argument. However, let's leave the code you have for now and we'll think about it again when we get to producing diffs.